app/Models/Booth.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+class Booth extends Model
+{
+    use HasFactory;
+
+    /**
+     * Establishes the connection
+     * A booth belongs to a team
```

Comment:
Stating the obvious: "Establishes the connection". Instead, describe it as an attribute. Suggestion: "The Team that owns this Booth"